CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 - Added `WindowBuilder::with_position` to allow setting the position of a `Window` on creation. Supported on Windows, macOS and X11.
 - Added `Window::drag_window`. Implemented on Windows, macOS, X11 and Wayland.
 - On X11, bump `mio` to 0.7.
+- On macOS, emit `RedrawRequested` events while the window is being resized.
```

Comment:
```suggestion
- On macOS, emit `RedrawRequested` events immediately while the window is being resized.
```
Based on the PR title, linked issues, and the tiny amount of code I've seen in the macOS backend, this seems like it describes the changes better.